hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/EvmTxProcessor.java

```diff
@@ -144,10 +117,26 @@ protected TransactionProcessingResult execute(
             final Account relayer) {
         final Wei gasCost = Wei.of(Math.multiplyExact(gasLimit, gasPrice));
         final Wei upfrontCost = gasCost.add(value);
-        final long intrinsicGas =
-                gasCalculator.transactionIntrinsicGasCost(Bytes.EMPTY, contractCreation);
 
-        final HederaWorldState.Updater updater = (HederaWorldState.Updater) worldState.updater();
+        final HederaTracer hederaTracer =
```

Comment:
nit: a comment here would be useful regarding the tracer setup check for static and contract action side cars